CMakeLists.txt

```diff
@@ -0,0 +1,376 @@
+cmake_minimum_required(VERSION 3.20)
+
+# Project definition
+project(SuperNOVAS
+    VERSION 1.5.0
```

Comment:
Note to self: I'll have to remember to update the version here too, for future releases.... :-)

Reply:
Don't worry. I don't think there is a catchall solution. The version appears in `novas.h`, which is the most important one. But then, it goes into the `Doxyfile` and `README.md` also (major.minor), and now `CMakeLists.txt`. Given I had to check/update in at least 3 places already, doing a fourth is not going to make a difference :-)